microprofile.jdt/org.eclipse.lsp4mp.jdt.test/src/main/java/org/eclipse/lsp4mp/jdt/core/faulttolerance/properties/MicroProfileFaultTolerancePropertiesTest.java

```diff
@@ -73,7 +73,7 @@ public void microprofileFaultTolerancePropertiesTest() throws Exception {
 
 				// <annotation>/<parameter>
 				p(null, "Bulkhead/value", "int",
-						"Specify the maximum number of concurrent calls to an instance. The value must be greater than 0. Otherwise, [org.eclipse.microprofile.faulttolerance.exceptions.FaultToleranceDefinitionException](jdt://contents/microprofile-fault-tolerance-api-2.0.3.jar/org.eclipse.microprofile.faulttolerance.exceptions/FaultToleranceDefinitionException.class?=microprofile-fault-tolerance/%5C/home%5C/jenkins%5C/.m2%5C/repository%5C/org%5C/eclipse%5C/microprofile%5C/fault-tolerance%5C/microprofile-fault-tolerance-api%5C/2.0.3%5C/microprofile-fault-tolerance-api-2.0.3.jar=/maven.pomderived=/true=/=/maven.groupId=/org.eclipse.microprofile.fault-tolerance=/=/maven.artifactId=/microprofile-fault-tolerance-api=/=/maven.version=/2.0.3=/=/maven.scope=/compile=/=/maven.pomderived=/true=/%3Corg.eclipse.microprofile.faulttolerance.exceptions%28FaultToleranceDefinitionException.class#28) occurs."
+						"Specify the maximum number of concurrent calls to an instance. The value must be greater than 0. Otherwise, org.eclipse.microprofile.faulttolerance.exceptions.FaultToleranceDefinitionException occurs."

```

Comment:
that's looks very fishy. It means the FaultToleranceDefinitionException type is not resolved by JDT and/or the Javadoc/markdown fails to be resolved as well.

Reply:
So we should be expecting a link? It didn't use to have a link here: https://github.com/eclipse/lsp4mp/commit/6a29828073e3ecbf952873c9e2854df1f9105b8d

Reply:
Using the built language server in VS Code, the link works as expected.

Reply:
In VS Code, using the latest released vscode-microprofile to open [microprofile-fault-tolerance](https://github.com/eclipse/lsp4mp/tree/master/microprofile.jdt/org.eclipse.lsp4mp.jdt.test/projects/maven/microprofile-fault-tolerance/), the link is missing from the Javadoc.

<img width="1076" alt="Screenshot 2024-07-08 at 16 42 51" src="https://github.com/eclipse/lsp4mp/assets/148698/e4ad584c-907a-47bc-be2d-04bac9d6eeaf">

JDT.LS itself renders the link properly:
<img width="824" alt="Screenshot 2024-07-08 at 16 47 16" src="https://github.com/eclipse/lsp4mp/assets/148698/e2fb7629-662d-45dc-a814-8afd151f329c">

So I think the problem is somewhere in lsp4mp's call to JDT.LS Javadoc rendering API

Reply:
With Snjezana's help, I've fixed the test so that the Markdown link is returned properly.